site/gatsby-site/src/components/cite/Tools.js

```diff
@@ -162,4 +198,123 @@ function Tools({
   );
 }
 
+function RemoveDuplicateModalContents({ incident }) {
+  const [duplicateIncidentId, setDuplicateIncidentId] = useState(null);
+
+  const [deletingDuplicate, setDeletingDuplicate] = useState(false);
+
+  const [updateIncident] = useMutation(UPDATE_INCIDENT);
+
+  const [insertDuplicate] = useMutation(INSERT_DUPLICATE);
+
+  const [updateClassifications] = useMutation(UPDATE_CLASSIFICATIONS_INCIDENT_ID);
+
+  const { data: duplicateIncidentData, loading: duplicateIncidentLoading } = useQuery(
+    FIND_INCIDENT,
+    { variables: { query: { incident_id: duplicateIncidentId } } }
+  );
+
+  const duplicateIncident = duplicateIncidentData?.incident;
+
+  return (
+    <>
+      <Trans>Transfer reports to incident</Trans>{' '}
+      <input
```

Comment:
What about using the existing `IncidentsField` component? It has auto-complete and will be more consistent with other forms, although I get that it is more geared toward multiple selections, we could add an option to limit selection to one item.